C-BackEnd/src/xcodeml/c/decompile/XcXmpFactory.java

```diff
@@ -59,10 +59,10 @@
      *
      * @return the name of temporary variable name.
      */
-    private static String _createTmpValueName()
+    /*private static String _createTmpValueName()
```

Comment:
Is it possible to keep this function for future use?

Reply:
Yes, I can revert this change.
Please note however, that currently it is not called anywhere in xcodeml-tools, so it's probably dead code.

Reply:
Ok. I'll restore it if we need it in the future.